src/components/Games/Katazauras/GameKatazauras.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect, useRef } from 'react';
+import Cat from './Cat';
+import Obstacle from './Obstacle';
+import obstacle1 from '@/images/katazaugasImages/obWall.png';
+import obstacle2 from '@/images/katazaugasImages/obPoop.png';
+import obstacle3 from '@/images/katazaugasImages/obVape.png';
+
+interface CatType {
+  x: number;
+  y: number;
+  dy: number; // for jumping and gravity
+  jumping: boolean;
+  jumpsLeft: number;
+}
+
+interface ObstacleType {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+  image: string;
+}
+
+const obstacleImages = [obstacle1, obstacle2, obstacle3];
+const GROUND_LEVEL = 320; // where lands
+const PATH_HEIGHT = 40;
+
+const Game: React.FC = () => {
+  const [cat, setCat] = useState<CatType>({
+    x: 50,
+    y: GROUND_LEVEL,
+    dy: 0,
+    jumping: false,
+    jumpsLeft: 2,
+  });
+  const [obstacles, setObstacles] = useState<ObstacleType[]>([]);
+  const [score, setScore] = useState<number>(0);
+  const gameInterval = useRef<number | null>(null); // starts as null and will hold a number later
+
+  useEffect(() => {
+    const updateGame = () => {
```

Comment:
Sita funkcija reiktu iskelti is useEffect, ir tada prie dependencies prideti `updateGame`